.github/workflows/release.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         with:
           # This expects you to have a script called release which does a build for your packages and calls changeset publish
           publish: pnpm release
+          version: changeset version && node ./packages/kit/scripts/update_version.js
```

Comment:
when the `Version Packages` PR get's created, the version info get's updated

Reply:
I could not test if the CI action actually works, but I would assume it does so

Reply:
I'd prefer to not use && in the version of release.yml and add a separate "version" script to package.json

Reply:
done